tests/roles/linux-system-roles.template

```diff
@@ -0,0 +1 @@
+../..
```

Comment:
This link should be called `linux-system-roles.certificate` - which is the fully qualified name of your role

Reply:
4df714a3abd44bbcabce5051b55ccdd76bf1ef5b